### PR TITLE
ADD: adding guidelines for communication

### PR DESCRIPTION
**Pull request by ghost**
This should help reduce the notifications, keep everything open to the markers, and they won't be able to argue evidence. Ideally this should also reduce confusion and address some things from the retro

### CODE REVIEW

**Comment by j-6** [comment]
"1. All announcements to general only "

Is it necessary to mention it's the 'Thoth Tech' General channel? There are a few General channels related to the unit/s, SIT374/SIT764 Team Project (A) ..., Deakin SIT Capstone Cohorts, project teams etc.

File: docs/communication/communication.md [hunk 1]
```diff
@@ -11,23 +11,28 @@ asynchronous communication as starting point.
...
 are written down and accessible to everyone in the company.
 
+## Guidelines
+
+1. All announcements to Thoth Tech general channel only _(clarity in notifications)_
+1. Use channels for all general to major discussion
+1. Use chat / direct message for private or personal discussion
+1. Use GitHub PRs for feedback on code or documentation _(building visibility)_
+1. Use Trello for planning and assigning tasks
+1. Use meetings for sychronous discussion _(for private channels, record and store the meeting in the channel files)_
```
Comment (tancnle) [inline docs/communication/communication.md]: **suggestion:** Small typo

```suggestion
1. Use meetings for synchronous discussion _(for private channels, record and store the meeting in the channel files)_
```

File: docs/communication/communication.md [hunk 2]
```diff
@@ -11,23 +11,28 @@ asynchronous communication as starting point.
...
 ## Tips and practices
 
-1. You are not expected to be available all the time. There is no expectation
-   to respond to messages outside of your planned working hours.
+1. You are not expected to be available all the time. There is no expectation to respond to messages outside of your planned working hours.
 1. All company data should be shareable by default.
-1. It is OK to ask as many questions as you have. Please do them in a way so
-   many people can answer them and many people see the answer. Use Pull Requests
-   or public chat channels instead of direct messages.
+1. It is OK to ask as many questions as you have. Please do them in a way so many people can answer them and many people see the answer. Use Pull Requests or public chat channels instead of direct messages.
```
Comment (tancnle) [inline docs/communication/communication.md:28]: **suggestion:** The maximum line length is set to 80 (it is set in the prettier [config](https://github.com/thoth-tech/handbook/blob/main/.prettierrc). We set a short line length to allow better readability, for example, preview split-screen or in code review. Could you kindly merge the changes from master `git merge master` to get the required config 🙏🏼 ?

**Review by tancnle — COMMENTED** [review]
Great work @talzid 👍🏼 I have some minor suggestions. Please let me know what you think.

File: docs/communication/communication.md (hunk 2 above)
Comment (ghost) [reply docs/communication/communication.md:28]: I am pretty sure I did this correctly, could you please double check as part of the review. Thank you

**Review by tancnle — APPROVED** [review]
Look great @talzid. I think it would be great to socialise this guide in our weekly announcement. What do you think?

**Comment by ghost** [comment]
Yes definitely @tancnle !
Also please action this by only sending the weekly announcement to the general channel 😄

**Pull request merged**